src/test/java/org/folio/edge/patron/MainVerticleTest.java

```diff
@@ -77,7 +77,7 @@ public class MainVerticleTest {
   private static final String itemId = UUID.randomUUID().toString();
   private static final String instanceId = UUID.randomUUID().toString();
   private static final String holdId = UUID.randomUUID().toString();
-  private static final String apiKey = ApiKeyUtils.generateApiKey(10, "diku", "diku");
+  private static final String apiKey = ApiKeyUtils.generateApiKey(10, "consortium", "consortium_admin_user");
```

Comment:
Please check if there is a need for a different apiKey for consortium as previously this apiKey was initialized for diku.

Reply:
reverted with earlier entry of diku